pwndbg/commands/strings.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+import argparse
+import os
+from typing import List
+import pwndbg
+import pwndbg.commands
+import pwndbg.aglib.memory
+from pwndbg.commands import CommandCategory
+
+import re
+
+from pwndbg.lib.memory import Page 
+
+if pwndbg.dbg.is_gdblib_available():
+    import gdb # type: ignore
```

Comment:
Is this needed? (cc: @mbrla0)

Reply:
Actually no, I accidentally left it. I fixed this unused import and some of trailing whitespaces in the newest [commit](https://github.com/pwndbg/pwndbg/pull/2505/commits/cb3fcbe6e297ff1fba5d0702647ad99ae4105781).